test/gui/shared/steps/steps.py

```diff
@@ -640,3 +640,11 @@ def step(context):
         'Ser&ver Address',
     )
     waitForObject(AccountConnectionWizard.SERVER_ADDRESS_BOX)
+
+
+@Then("the following tabs in the toolbar should match the default baseline")
+def step(context):
+
+    for tabName in context.table:
+        print(tabName)
```

Comment:
do we need this print?

Reply:
No we don't need that it was added just for debug purpose. Now it's removed.